UI/src/view/PointAssetLayer.js

```diff
@@ -229,6 +229,10 @@
       eventListener.listenTo(eventbus, layerName + ':unselected', handleUnSelected);
       eventListener.listenTo(eventbus, layerName + ':changed', handleChanged);
       eventListener.listenTo(eventbus, 'application:readOnly', toggleMode);
+      eventListener.listenTo(eventbus, 'toggleWithRoadAddress', function(){
+        if(applicationModel.getSelectedLayer() == layerName)
+          me.refreshView();
+      });
```

Comment:
In this case, it's more pretty if we create a function bellow and not at eventListener, only for use the same idea the others "listenTo's".